src/main/java/org/broadinstitute/dsm/db/dao/kit/BSPKitDao.java

```diff
@@ -0,0 +1,231 @@
+package org.broadinstitute.dsm.db.dao.kit;
+
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.db.SimpleResult;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.InstanceSettings;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.db.dao.Dao;
+import org.broadinstitute.dsm.db.dao.ddp.kitrequest.KitRequestDao;
+import org.broadinstitute.dsm.model.KitDDPNotification;
+import org.broadinstitute.dsm.model.Value;
+import org.broadinstitute.dsm.model.bsp.BSPKitInfo;
+import org.broadinstitute.dsm.model.bsp.BSPKitQueryResult;
+import org.broadinstitute.dsm.model.bsp.BSPKitStatus;
+import org.broadinstitute.dsm.statics.ApplicationConfigConstants;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.broadinstitute.dsm.statics.ESObjectConstants;
+import org.broadinstitute.dsm.util.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Response;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.*;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+
+public class BSPKitDao implements Dao<BSPKitDao> {
```

Comment:
Dao needs to return Dto(or Dto like class which holds data fetched from DB) not itself.